language-support/hs/bindings/README.md

```diff
@@ -21,21 +21,55 @@ You can find some usage examples
 [here](/language-support/hs/bindings/test/DA/Ledger/Tests.hs) and
 [here](/language-support/hs/bindings/examples/chat/src/DA/Ledger/App/Chat/ChatLedger.hs).
 
-Using these bindings **outside of this repository** can be a bit
-tricky since there are quite a few packages required that are not
-published to Hackage at the moment.
-
-To make things a bit easier there is a `build_packages.sh` script in
-this directory that you can run as follows: `./build_packages.sh
-TARGET_DIR`.  This will build the tarballs for all cabal packages and
-put them in `TARGET_DIR`. If `TARGET_DIR` does not already contain a
-`cabal.project` file, it will also create a `cabal.project` file that
-references the created tarballs so all dependencies should be resolved
-properly. You will also need to install gRPC 1.22, see
-https://github.com/grpc/grpc/blob/master/BUILDING.md for installation
-instructions. If you install it in a non-standard location, you need
-to adjust extra-lib-dirs and extra-include-dirs for
-grpc-haskell-core. The default `cabal.project` file contains an
-example of how to do this.
-
-The main package for the ledger bindings is called `daml-ledger`.
+
+# Using these bindings **outside of this repository**
+
+*Note: These instructions document the current process. We do plan make it simpler!*
+
+These bindings can be build as a standalone Haskell package `daml-ledger` using `stack`. Currently, `stack` references the code directly in the `daml` repo. The only annoyance is that part of the Haskell code for `daml-ledger` is generated from `.proto` files, and this generation step must be performed using the `bazel` build. We plan to simplify this step by directly providing `daml-ledger` as a package on hackage.
+
+As well as `stack`, the only other prerequisite is to have `grpc` installed.
+We are currently using gRPC version `1.22.1`. To install `grpc`  requires building `grpc` from source (really!). See detailed instructions here: https://github.com/grpc/grpc/blob/master/BUILDING.md.
+
+Note, the `grpc` build instructions warn against doing a global install:
+
+> *"WARNING: After installing with make install there is no easy way to uninstall, which can cause issues if you later want to remove the grpc and/or protobuf installation or upgrade to a newer version."*
+
+If you follow this advice, then you must adjust `extra-lib-dirs` in your `stack.yaml` config. This is explicated in the detailed instructions below.
+
+
+## Download and build grpc at version 1.22.1
+
+    cd ~/github
+    git clone -b $(curl -L https://grpc.io/release) https://github.com/grpc/grpc
+    cd grpc
+    git checkout v1.22.1
+    git submodule update --init
+    make
+
+## Clone daml repo; force Haskell code generation
+
+    cd ~/github
+    git clone https://github.com/digital-asset/daml.git
+    cd daml
+    direnv allow
```

Comment:
most people will not have `direnv` installed, so probably better to just enale the dev-env once as we do on CI
```suggestion
    bash
    eval $(dev-env/bin/dade-assist)
```
(extra `bash` to create a new shell, so they can then undo the `dade assist` call by typing exit; it may be worth explaining that in text around the code block)

Reply:
I tried this, but I still needed `direnv allow`.

Reply:
so I didn't make the change you suggested